src/client/util.ts

```diff
@@ -22,6 +22,15 @@ export class Util {
             message: "Member is not part of message",
         };
 
+        const disabledCommands = process.env.COMMANDS_DISABLED?.split(',');
+
+        if (disabledCommands?.find(x => x == name)) {
+            return {
```

Comment:
When this returns, what it should do is reply to the user also, to say its disabled